cloudwash/providers/aws.py

```diff
@@ -19,7 +24,10 @@ def cleanup(**kwargs):
         dry_data['VMS']['skip'] = []
         for items in data:
             dry_data[items]['delete'] = []
-        with compute_client("aws", aws_region=region) as aws_client:
+
+        # Differentiate between the cleanup region to the client region, if client_region defined
+        r = client_region or region
+        with compute_client("aws", aws_region=r) as aws_client:
```

Comment:
The client_region should never be used for fetching/cleaning reasons. That must come from regions setting.

Reply:
ditto